modules/network/basic-cdk/stack.py

```diff
@@ -12,6 +12,9 @@
 
 _logger: logging.Logger = logging.getLogger(__name__)
 
+VPC_CIDR = "10.0.0.0/16"
+SUBNET_SIZE = 24
```

Comment:
Shall we add input parameters for these then?

Reply:
I thought about it, but at this point of time or even in the future, i see only us using the networking module to deploy the required networking. Ideally customers will use other mechanisms such as LZs etc

Reply:
Ok makes sense